sdk/java-sdk/src/main/scala/kalix/javasdk/impl/GrpcClients.scala

```diff
@@ -90,6 +90,9 @@ final class GrpcClients(system: ExtendedActorSystem) extends Extension {
     }
   }
 
+  def getProxyGrpcClient[T](serviceClass: Class[T]): T =
+    getGrpcClient(serviceClass, "localhost", 9000)
```

Comment:
localhost as we talked about?

Reply:
Yep, that's what I was mentioning today and that's what I mean that I needed to change something in the SDK.